tests/integrationTests/TestBeaconPODSending.js

```diff
@@ -116,24 +116,38 @@ function validateTrackedBeaconsFile() {
 let DoneMethod;
 describe('test full flow', function () {
 	it('test - Beacon POD Sending', function (done) {
-		this.timeout(300000);
+		this.timeout(150000);
 		DoneMethod = done;
 		if (fs.existsSync(beaconTrackingFile))
 			fs.unlinkSync(beaconTrackingFile);
 		playServerTestingHelper.initTestHelper(serviceUrl, impersonatePartnerId, secretImpersonatePartnerId);
 		playServerTestingHelper.initClient(playServerTestingHelper.serverHost, playServerTestingHelper.partnerId, playServerTestingHelper.adminSecret, testInit);
 	});
 });
+
+let entry;
```

Comment:
i guess you are initiating the variable here so that finish test can recognize it , same as you saw the DoneMethod be declared - this is c programming , can't it be initialized as class member or even if not -place it as a global variable